src/coreclr/vm/codeman.cpp

```diff
@@ -1385,6 +1388,12 @@ void EEJitManager::SetCpuInfo()
                                         if ((cpuidInfo[EBX] & (1 << 5)) != 0)                               // AVX2
                                         {
                                             CPUCompileFlags.Set(InstructionSet_AVX2);
+
+                                            __cpuidex(cpuidInfo, 0x00000007, 0x00000001);
+                                            if ((cpuidInfo[EAX] & (1 << 4)) != 0)                           // AVX-VNNI
```

Comment:
We should double check the behavior of this bit on non-Intel platforms (namely AMD) to ensure:
1. It isn't actively in use
2. The platform returns `0`, not `1`

We've seen at least one case (`ERMSB`) in the past where the bit was not officially in use but returned `1` rather than zero and where it needed special casing.

Reply:
At first glance (http://support.amd.com/TechDocs/24594.pdf), AMD only supports `__cpuidex(cpuidInfo, 0x00000007, 0x00000000);`.

I'll explicitly query the bit my 5950X later today to validate its zero as well.

Reply:
@tannergooding Does this work correctly on that system?

Reply:
Yes. All values return `0` and so this will correctly report `false`.